modules/doc/content/training/index.md

```diff
@@ -4,18 +4,28 @@
 Upcoming MOOSE related training in chronological order:
 !style-end!
 
-- [ART/NEAMS M&S Molten Salt Reactors @PHYSOR24 (2024 April 21)](more_detail/PHYSOR_MS_2024_04_21.md)
-- [ART/NEAMS M&S Pebble Bed Reactors @PHYSOR24 (2024 April 21)](more_detail/PHYSOR_PB_2024_04_21.md)
 - [MOOSE Framework Fundamentals (2024 June 4-6, Idaho Falls)](more_detail/MOOSE_2024_06_04-06_IF.md)
 - [MOOSE Framework Fundamentals (2024 October 15-17, University of Illinois Urbana-Champaign)](more_detail/MOOSE_2024_10_15-17_UIUC.md)
 
-Past Training:
+# Conferences
+
+!style! halign=left
+Upcoming MOOSE related Conferences:
+!style-end!
+
+- MOOSE Conference (2025 March 10-14, Idaho Falls) - Registration link coming soon.
+
+#### Past Training:
+
+- ART/NEAMS M&S Molten Salt Reactors @PHYSOR24 (2024 April 21)
+
+- ART/NEAMS M&S Pebble Bed Reactors @PHYSOR24 (2024 April 21)
```

Comment:
@milljm I moved the PHYSOR trainings to "past training" in #27490, which is now in master branch, but it seems like GitHub isn't complaining? I guess it can find a good merge solution.

Reply:
hmm, probably because I added a Conference section that may have ended up on these lines. Still, I'll remove these changes and rebase when I do the justification change on the landing image.